diff --git a/deliver-imap.c b/deliver-imap.c
index 0299ba1..0b637e1 100644
--- a/deliver-imap.c
+++ b/deliver-imap.c
@@ -216,15 +216,10 @@ deliver_imap_deliver(struct deliver_ctx *dctx, struct actitem *ti)
 retry:
 	/*
 	 * Determine the mail size, not forgetting lines are CRLF
-	 * terminated. The Google IMAP server is written strangely, so send
-	 * the size as if every CRLF was a CR if the server has XYZZY.
+	 * terminated.
 	 */
 	count_lines(m, &total, &body);
 	maillen = m->size + total - 2;
-	if (fdata.capa & IMAP_CAPA_XYZZY) {
-		log_debug2("%s: adjusting size: actual %zu", a->name, maillen);
-		maillen = m->size;
-	}
 
 	/* Send an append command. */
 	if (imap_not_clean(folder)) {
